_posts/2020-04-28-archivesspace-cleanup-dates.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "ArchivesSpace Clean Up: Adding Structured Dates to an Entire Repository"
+date: 2020-04-28T11:30:00
+author: Amy Berish
+layout: post
+categories:
+  - Data Cleanup
+  - Collaboration
+tags:
+  - data
+  - description
+  - discovery
+  - learning
+excerpt_separator: <!--more-->
+---
+
+As you may already know, the Digital Strategies Team is working on developing and integrating a new discovery system for the RAC. In preparation for this new system, we have been working on [cleaning up data in ArchivesSpace] (https://blog.rockarch.org/archivesspace-clean-up-an-outline) so that the new system can more effectively point researchers to relevant material. My involvement in the data cleanup phase included adding structured (begin/end) dates to all archival objects in our ArchivesSpace repository. In this post, I’ll be talking about how we developed a semi-automated plan to clean up date expressions and add structured dates to our repository by parsing date expressions.
+<!--more-->
+
+## A Bad First “Date”
+Our initial idea to improve discovery was to facilitate faceted date searching by populating structured (begin/end) dates for all series-level components in our ArchivesSpace repository. At first, we thought to use a built-in feature in ArchivesSpace called “Calculate Dates” which automatically generates an accurate date range at a collection, series, or subseries level of a finding aid. Unfortunately, we were unable to use this tool because it relies on the existence of structured dates on archival objects below it and in most cases, those did not exist.
```

Comment:
```suggestion
Our initial idea to improve discovery was to facilitate faceted date searching by populating structured (begin/end) dates for all series-level components in our ArchivesSpace repository. At first, we thought to use a built-in feature in ArchivesSpace called “Calculate Dates” which automatically generates an accurate date range at a collection, series, or subseries level of a finding aid. Unfortunately, we were unable to use this tool because it relies on the existence of structured dates on archival objects below it, and in most cases, those did not exist.
```